exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -88,10 +91,19 @@ func newPRWExporter(cfg *Config, set exporter.CreateSettings) (*prwExporter, err
 
 // Start creates the prometheus client
 func (prwe *prwExporter) Start(ctx context.Context, host component.Host) (err error) {
-	prwe.client, err = prwe.clientSettings.ToClient(host, prwe.settings)
+	retryClient := retryablehttp.NewClient()
+	retryClient.HTTPClient, err = prwe.clientSettings.ToClient(host, prwe.settings)
 	if err != nil {
 		return err
 	}
+
+	// Configure retry settings
+	// Default settings reference - https://github.com/hashicorp/go-retryablehttp/blob/571a88bc9c3b7c64575f0e9b0f646af1510f2c76/client.go#L51-L53
+	retryClient.RetryWaitMin = 100 * time.Millisecond
+	retryClient.RetryWaitMax = 1 * time.Second
+	retryClient.RetryMax = 3
```

Comment:
why do you need to change the defaults? if the reason is for the unit tests to execute faster, please expose these settings in the prw struct and change them in the unit tests instead.

Reply:
If we expose this through configuration we should look to utilize the same structure and attributes as the Prometheus [`remote_write.queue_config`](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write), i.e., `min_backoff` and `max_backoff`.